mlflow/tracking/client.py

```diff
@@ -180,7 +316,34 @@ def create_experiment(self, name, artifact_location=None):
         :param name: The experiment name. Must be unique.
         :param artifact_location: The location to store run artifacts.
                                   If not provided, the server picks an appropriate default.
-        :return: Integer ID of the created experiment.
+        :return: String as an integer ID of the created experiment.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            # Create an experiment with a name that is unique and case sensitive.
```

Comment:
How about using the same explanation as the docstring of `create_experiment` ([link](https://github.com/mlflow/mlflow/pull/3489/files#diff-581b8f8c85a274ab9d0317aec207b364R634)):
```suggestion
            # Create an experiment with a name that must be unique and case sensitive.
```

?